### PR TITLE
Remove alara params decay times

### PR DESCRIPTION
**Pull request by zxkjack123**
Users need to define decay times in both `config.ini` and `alara_params.txt` for running r2s before this PR.
This PR keeps the decay times definition in "config.ini" and remove them from the "alara_params.txt".  
By the way, some names of `alara_geom` are changed into `alara_inp`.
Major changes:
- Remove decay times definition from `alara_params.txt`
- Change`alara_geom` to `alara_inp` at some places

### CODE REVIEW

File: pyne/r2s.py [hunk 1]
```diff
@@ -14,9 +14,9 @@
 def irradiation_setup(flux_mesh, cell_mats, alara_params, tally_num=4,
                       geom=None, num_rays=10, grid=False, flux_tag="n_flux",
                       fluxin="alara_fluxin", reverse=False,
-                      alara_inp="alara_geom", alara_matlib="alara_matlib",
+                      alara_inp="alara_inp", alara_matlib="alara_matlib",
                       output_mesh="r2s_step1.h5m", output_material=False,
-                      sub_voxel=False):
+                      decay_times=['1 s'], sub_voxel=False):
```
Comment (scopatz) [inline pyne/r2s.py]: You should not put mutable containers (such as a list) as the default value of a kwarg.  Please use a tuple instead.

File: pyne/r2s.py [hunk 2]
```diff
@@ -111,6 +113,14 @@ def irradiation_setup(flux_mesh, cell_mats, alara_params, tally_num=4,
     record_to_geom(m, cell_fracs, cell_mats, alara_inp, alara_matlib,
                    sub_voxel=sub_voxel)
 
+    # write decay times into alara_inp
+    decay_str = 'cooling\n'
+    for dc in decay_times:
+        decay_str = ''.join([decay_str,'    ', dc, '\n'])
```
Comment (scopatz) [inline pyne/r2s.py]: PEP8: single space after comma

**Comment by scopatz** [comment]
Thanks for putting this in.  Generally looks good.  Couple of small issues.

**Comment by gonuke** [comment]
Was this actually a bug previously?  it seems that the decay times wouldn't necessarily match between different steps of R2S before...

**Comment by zxkjack123** [comment]
@gonuke . No, it wasn't a bug. But users had to define the cooling time two times, in both `alara_params.txt` and `config.ini`. However, they are actually the same information. So, I removed the definition of cooling time in `alara_params.txt`. Therefore, we can define the cooling time only once now (in `config.ini`).

**Comment by gonuke** [comment]
Thanks @zxkjack123

**Pull request merged**